openqa-label-known-issues-and-investigate-hook

```diff
@@ -6,21 +6,22 @@ set -eo pipefail
 # parameter and forwarding a complete job URL to "openqa-label-known-issues"
 # on stdin and all left unknowns to "openqa-investigate"
 
-id="${1:?"Need 'job_id'"}"
 host="${host:-"openqa.opensuse.org"}"
 scheme="${scheme:-"https"}"
 host_url="$scheme://$host"
-dirname=$(dirname "$0")
-script_dir=${script_dir:-$dirname}
 
 investigate-and-bisect() {
     local rc=0 test
     read -r test
-    echo "$test" | "$script_dir/openqa-investigate" || rc=$?
+    echo "$test" | "$(dirname "$0")"/openqa-investigate || rc=$?
```

Comment:
It would make sense to still use the `$script_dir` variable so `dirname` needs to be invoked only once.

Reply:
The main point is, I can't keep it as it is. I can't use a path to the script because I want to override it via functions in the test.
And that's also what other people can do if they want a different path.

I'm not sure how often I have the use case that I want to run this hook but the scripts from a certain directory.
In the beginning, when you wrote the hook, it was just `"$(dirname "$0")"/openqa-investigate`.

Reply:
> It would make sense to still use the `$script_dir` variable so `dirname` needs to be invoked only once.

I fixed by setting `dir` at the beginning, and adding it to `PATH`.